package.json

```diff
@@ -10,8 +10,8 @@
   },
   "engines": {
     "node": "^8.9.1",
```

Comment:
Can we change the node engine to be more lenient to newer versions of node too?

Reply:
Sure! Do we want to upgrade to Node 10 or stay at 8, just newer?

Reply:
I think we should make it relaxed enough to accept any engines that work with our build. So perhaps `>= 8.9.1 < 11`?

Reply:
@petebacondarwin We don't have a way to verify if it keeps working at a version we don't work on. Also, `.nvmrc` needs to specify one version, we can't use 2 at the same time. So if we want to allow newer versions maybe it'd be better to say `>=8.12.0` so that all newer versions are accepted?

Reply:
Happy to go with `>=8.12.0`